FinancialTracker/Models/Category.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Income+Category.swift
+//  FinancialTracker
+//
+//  Created by Boyan Pavlov on 9.02.22.
+//
+
+import UIKit
+
+protocol Category {
+    var getRawValue: String { get }
+}
+
+enum ExpenseCategory: String, CaseIterable, Category, Codable {
+    case transport = "Transport"
+    case grocery = "Grocery"
+    
+    case taxes = "Taxes"
+    case utility = "Utility"
+    case travel = "Travel"
+    
+    case other = "Other"
+    
+    var getRawValue: String {
+        return self.rawValue
```

Comment:
I do not see the point of `getRawValue`? Actually, it is obviously not necessary. 
Same with 
```
protocol Category {
     var getRawValue: String { get }
 }
```